opentargets.json

```diff
@@ -442,6 +416,19 @@
             "molecule_name",
             "molecule_type"
           ]
+        },
+        "target" : {
+          "properties" :{
+            "complex_id": {
+              "type": "string",
+              "description": "A ChEMBL protein complex identifier",
+              "pattern": "^CHEMBL[0-9]+$"
```

Comment:
@afaulconbridge why is this one limited to CHEMBL? The drug.id itself is not (see https://github.com/opentargets/json_schema/pull/72/files#diff-8733dc6847a1a17a6cfa055eb8a14605R383).

Reply:
Also, why should a target always be a protein complex? Can't it be a single protein and therefore an ENSEMBL or Uniprot code?